assets/scripts/segments/SegmentForPalette.jsx

```diff
@@ -127,17 +129,22 @@ function SegmentForPalette (props) {
         width: actualWidth * TILE_SIZE * PALETTE_SEGMENT_MULTIPLIER + 'px'
       }}
       className="segment segment-in-palette"
-      onPointerOver={handlePointerOver}
       data-testid="segment-for-palette"
     >
-      <SegmentCanvas
-        actualWidth={actualWidth}
-        type={props.type}
-        variantString={props.variantString}
-        randSeed={props.randSeed}
-        multiplier={PALETTE_SEGMENT_MULTIPLIER}
-        groundBaseline={PALETTE_GROUND_BASELINE}
-      />
+      <Tooltip target={props.tooltipTarget} label={getLabel(props)}>
```

Comment:
im not fully groking why Tooltip is in here twice? Once for disabled components and once for enabled?

Reply:
Disabled and enabled components have separate rendering paths, and they both must be individually wrapped with `<Tooltip />`.